app/server/appsmith-plugins/mysqlPlugin/src/test/java/com/external/plugins/MySqlPluginTest.java

```diff
@@ -54,14 +55,16 @@
 
     @SuppressWarnings("rawtypes") // The type parameter for the container type is just itself and is pseudo-optional.
     @ClassRule
-    public static MySQLContainer mySQLContainer = new MySQLContainer("mysql:5.7")
+    public static MySQLContainer mySQLContainer = new MySQLContainer(
+            DockerImageName.parse("mysql/mysql-server:latest-aarch64").asCompatibleSubstituteFor("mysql"))
```

Comment:
I think the test is failing in CI because we are forcing the arm64 architecture here. Can we use `latest` tag or `8.0.25` instead? This will peg our tests to a single version and make them more predictable and it also has both architectures available.

![Screenshot 2021-06-04 at 12 01 46](https://user-images.githubusercontent.com/120119/120756249-a573e180-c52c-11eb-8a12-378b2805b725.png)

```suggestion
            DockerImageName.parse("mysql/mysql-server:8.0.25").asCompatibleSubstituteFor("mysql"))
```

Reply:
Yes. This makes sense. I agree with you. I will make this change. For the SQL Server edge i couldn't find any release notes on the docker hub page.  They just have one Repo tag **latest**. 

Is there any way to get the past release version ? I tried checking the history of the image locally but no luck either.

Reply:
Looks like they have a few versions, as seen from <https://mcr.microsoft.com/v2/azure-sql-edge/tags/list>. Try with the `1.0.3`, if it works, we'll keep it, otherwise we'll continue with `latest`. Not much we can do if the version tags aren't provided by Microsoft.